host/sgx/calls.c

```diff
@@ -684,18 +745,19 @@ oe_result_t oe_ecall(
 /*
 **==============================================================================
 **
-** oe_call_enclave_function()
+** oe_call_enclave_function_by_table_id()
 **
-** Call the enclave function specified by the given function-id.
+** Call the enclave function specified by the given table-id and function-id.
 ** Note: Currently only SGX style marshaling is supported. input_buffer contains
```

Comment:
Actually, this comment is outdated. We support marshalling for both SGX and TrustZone.

Reply:
Thanks Anand. I pushed a change to remove that outdated comment.